cvat/apps/documentation/user_guide.md

```diff
@@ -359,406 +576,523 @@ Go to specified frame. Press ``~`` to highlight element.
+If a frame contains too many objects and it is difficult to annotate them
+due to many shapes that are placed mostly in the same place, it makes sense
+to lock them. Shapes for locked objects are transparent, and it is easy to
+annotate new objects. Besides, you can't change previously annotated objects
+by accident. Shortcut: ``L``.
 
 ![](static/documentation/images/image066.jpg)
 
 ## Interpolation mode (advanced)
 
 Basic operations in the mode were described above.
 
-Bounding boxes created in the mode have extra navigation buttons.
-- These buttons help to jump to previous/next key frame.
+Bounding boxes that were created in the mode, have extra navigation buttons.
+-   These buttons help to jump to previous/next key frame.
 
     ![](static/documentation/images/image056.jpg)
 
-- The button helps to jump to initial frame for the object (first bounding box for the track).
+-   The button helps to jump to initial frame for the object (first bounding box for the track).
 
     ![](static/documentation/images/image057.jpg)
 
-
 ## Attribute annotation mode (advanced)
 
 Basic operations in the mode was described above.
 
-It is possible to handle many objects on the same frame in the mode.
+It is possible to handle lots of objects on the same frame in the mode.
 
 ![](static/documentation/images/image058.jpg)
 
-It is more convenient to annotate objects of the same type. For the purpose it is possible to specify a corresponding filter. For example, the following filter will hide all objects except pedestrians: ``pedestrian``.
+It is more convenient to annotate objects of the same type. You can specify
+the appropriate filter in this case. For example, the following filter will
+hide all objects except pedestrians: ``pedestrian``.
 
-To navigate between objects (pedestrians in the case) use the following shortcuts:
+To navigate between objects (pedestrians in the case), use the following shortcuts:
 - ``Tab`` — go to the next object
 - ``Shift+Tab`` — go to the previous object.
 
-By default in the mode objects are zoomed in to full screen. Check
+By default, objects in the mode are zoomed in to full screen. Check
 ``Open Menu`` —> ``Settings`` —> ``AAM Zoom Margin`` for adjust that.
 
 ## Annotation with polygons
 
 It is used for semantic / instance segmentation.
 
-Be sure ``Z-Order`` flag in ``Create task`` dialog is enabled if you want to annotate polygons. Z-Order flag defines order of drawing. It is necessary to get right annotation mask without extra work (additional drawing of borders). Z-Order can be changed by `+`/`-` which set maximum/minimum z-order respectively.
+Be sure ``Z-Order`` flag in ``Create task`` dialog is enabled if you want to
+annotate polygons. Z-Order flag defines order of drawing. It is necessary to
+get right annotation mask without extra work (additional drawing of borders).
+Z-Order can be changed by ``+``/``-`` which set maximum/minimum z-order
+accordingly.
 
 ![](static/documentation/images/image074.jpg)
```

Comment:
The image contains "task creation" dialog with unchecked z-order. Is it OK? Instead "use LFS" is checked.

Reply:
I would prefer to have here a screenshot of intersected polygons to demonstrate why we need z-order at all.